src/_designs/components/labelType/index.vue

```diff
@@ -31,10 +31,11 @@ export default {
   @import "~@aeternity/aepp-components-3/src/styles/variables/colors";
   .label-wrapper {
     display: table-caption;
-    padding: 5px;
+    padding: 0.6rem;
     text-transform: uppercase;
```

Comment:
I also just norticed that the font size is wrong for this. It should be `@extend %face-uppercase-xs;`.
Then you can skip some of the extra styling as well.

Reply:
done